ui/src/components/info-page/terms-of-service-page/terms-of-service.md

```diff
@@ -0,0 +1,63 @@
+# **Terms and Conditions (August 1, 2024)**
```

Comment:
Perhaps BE should return this? I think fine to keep as a FE asset for now though.

Reply:
I like that it's in the source code actually, at least for now.